synapse/config/registration.py

```diff
@@ -330,6 +334,19 @@ def generate_config_section(self, generate_secrets=False, **kwargs):
             #email: https://example.com     # Delegate email sending to example.com
             #msisdn: http://localhost:8090  # Delegate SMS sending to this local process
 
+        # If enabled, don't let users set their own display names/avatars
+        # other than for the very first time (unless they are a server admin).
+        # Useful when provisioning users based on the contents of a 3rd party
+        # directory and to avoid ambiguities.
+        #
+        #disable_set_displayname: false
```

Comment:
We usually try to avoid double negations in our configuration file to reduce confusion over what an option means, could you please make it `enable_[...]` instead of `disable_[...]` for all three options?